lib/elastic_apm/normalizers/grape/endpoint_run.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module ElasticAPM
+  module Normalizers
+    module Grape
+      # @api private
+      class EndpointRun < Normalizer
+        register 'endpoint_run.grape'
+
+        TYPE = 'app'
+        SUBTYPE = 'resource'
+
+        FRAMEWORK_NAME = 'Grape'
+
+        def normalize(transaction, _name, payload)
+          transaction.name = endpoint(payload[:env])
+          unless transaction.config.framework_name == FRAMEWORK_NAME
```

Comment:
Do we need to check beforehand? If we were to get this twice (when would we?) it can set the same value twice, no worries 😊

Reply:
Oh, it's checking against the config, I see. Sorry. Then it's fine to check as we'll same a few chars per transaction payload 👍

Reply:
I only want to add the transaction-level service info if it should differ from the metadata values.